### DISCUSSION TITLE
How can I reload a mod with command?

### DISCUSSION THREAD

**Discussion by MuscularPuky** [ramensoftware/windhawk#162] [Q&A]
until now, on my program made for Windhawk, I have implementd by toggle the mod with registry. now I'm refactoring lotsa spaghetti, so fixed lots of bugs. but one things, previous method, toggling the mod with registry causes a hundred of ms delay. I wanna slick it like when I'm saving the mod config on Windhawk. that cool doesn't make any delay. is there any command or method to reload the moad? I tried to find Windhawk parameter but was no luck.

**Comment by m417z** [comment]
Windhawk does the same - it just modifies the `Disabled` registry value.
